components/camel-google/camel-google-bigquery/src/main/java/org/apache/camel/component/google/bigquery/GoogleBigQueryConfiguration.java

```diff
@@ -59,7 +59,7 @@ public void parseRemaining(String remaining) {
         projectId = parts[0];
         datasetId = parts[1];
         if (parts.length > 2) {
-            tableId = parts[3];
+            tableId = parts[2];
```

Comment:
Oops. My bad on this one. I was trying to see if the tests would catch this but I forgot to fix it. Great catch, thanks!!!

Reply:
Np. I upgraded test to catch it.